spec/support/request/web_helper.rb

```diff
@@ -128,9 +128,11 @@ def close_select2
   end
 
   def tomselect_search_and_select(value, options)
-    page.find("[name='#{options[:from]}']").sibling(".ts-wrapper").click
-    page.find(:css, '.ts-dropdown input.dropdown-input').set(value)
-    page.find(:css, '.ts-dropdown .ts-dropdown-content .option', text: value).click
+    tomselect_wrapper = page.find("[name='#{options[:from]}']").sibling(".ts-wrapper")
+    tomselect_wrapper.find(".ts-control").click
```

Comment:
tom select along with capybara was behaving weird, and I was seeing this failure. So I did multiple clicks. I hope it is ok.:
```
Capybara::ElementNotFound:
       Unable to find visible css ".ts-dropdown input.dropdown-input" within #<Capybara::Node::Element tag="div" path="//HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[4]/DIV[1]/DIV[1]/DIV[1]/DIV[2]/DIV[2]/DIV[1]/FORM[1]/DIV[1]/DIV[1]">
```